misc/snapshotter/Dockerfile

```diff
@@ -26,6 +26,14 @@ COPY containerd-nydus-grpc /usr/local/bin/
 RUN chmod +x /usr/local/bin/containerd-nydus-grpc
 COPY nydusd-config.fusedev.json /etc/nydus/config.json
 COPY nydusd-config-localfs.json /etc/nydus/localfs.json
+COPY nydusd-config.fusedev.json /etc/nydus/nydusd-fusedev.json
+COPY nydusd-config-localfs.json /etc/nydus/nydusd-localfs.json
+COPY nydusd-config.fscache.json /etc/nydus/nydusd-fscache.json
+COPY config-blockdev.toml /etc/nydus/config-blockdev.toml
+COPY config-proxy.toml /etc/nydus/config-proxy.toml
+COPY config-fscache.toml /etc/nydus/config-fscache.toml
+COPY config-fusedev.toml /etc/nydus/config-fusedev.toml
+
 COPY entrypoint.sh /
 
```

Comment:
Regarding the renaming of [config-coco-host-sharing.toml → misc/snapshotter/config-blockdev.toml](https://github.com/containerd/nydus-snapshotter/pull/570/commits/25a78181551081d3d28466044ec9f20379d15065#diff-8d4453f632ff895accc5ef1ef82003c8cc5cce096ee1f2db46006318ee0f99fb) and [...napshotter/config-coco-guest-pulling.toml → misc/snapshotter/config-proxy.toml](https://github.com/containerd/nydus-snapshotter/pull/570/commits/25a78181551081d3d28466044ec9f20379d15065#diff-62f9dc5bb062667d6615355d70c1df51ade718500252ce6e1788e14a99be9a54) , would make sense to add the `kata-` filename prefix because they should be used with kata containers?